api/src/main/resources/workbench-api.yaml

```diff
@@ -12551,6 +12551,92 @@ components:
           type: string
           description: User-reported races information, comma-delimited if multiple
             answers were provided.
+        dsv2CompletionTime:
```

Comment:
All fields are prefixed with "dsv2" in order to represent that it comes from the demographic survey v2. It felt duplicative to add that to all 25+ dsv2 fields, but I am open to suggestions.